
bag_output: "/home/stew/data/2019-09-05_callan_park_big_loop/callan_park_localiser.bag"
bag_input: "/home/stew/data/2019-09-05_callan_park_big_loop/2019-09-05-15-44-53_callan_park_big_loop.bag"
